src/TraceEvent/TraceEvent.cs

```diff
@@ -194,6 +194,23 @@ public KernelTraceEventParser Kernel
             }
         }
 
+        /// <summary>
+        /// For convenience, we provide a property returns a ProcessMetadataTraceEventParser that knows 
+        /// how to parse all the Process Metadata events into callbacks.
+        /// </summary>
+        public ProcessMetadataEventSourceTraceEventParser ProcessMetadata
```

Comment:
Similar to the comment above, another spot where the name is currently showing up in public API surface